LCM/dsc/engine/ModuleLoader/ModuleLibrary/ModuleHandler.c

```diff
@@ -469,7 +469,6 @@ MI_Result GetModuleLoader( _In_ MI_Application *miApp,
 #if defined(BUILD_OMS)
     if (g_DscHost == MI_FALSE)
     {
-#endif
```

Comment:
This change was indroduced in 2019 as a part of Drop OMI Dependency and this was causing any OMI call to fail with 'FindFirstFile Failed'.